pkgs/node/src/chain.zig

```diff
@@ -179,11 +179,40 @@ pub const BeamChain = struct {
             return;
         };
 
-        self.logger.info("chain received on slot cb at slot={d} head={any} headslot={d}", .{
-            //
+        // Get additional chain information
+        const justified = self.forkChoice.fcStore.latest_justified;
+        const finalized = self.forkChoice.fcStore.latest_finalized;
+
+        // Calculate chain progress
+        const blocksBehind = if (slot > fcHead.slot) slot - fcHead.slot else 0;
```

Comment:
snake case variable is best practical follow the guide https://ziglang.org/documentation/0.14.1/#Style-Guide